common-npm-packages/azure-arm-rest-v2/azure-arm-common.ts

```diff
@@ -296,7 +296,17 @@ export class ApplicationTokenCredentials {
         // proxy usage
         const agentProxyURL = tl.getVariable("agent.proxyurl") ? new URL(tl.getVariable("agent.proxyurl")) : null;
         const agentProxyBypassHosts = tl.getVariable("agent.proxybypasslist") ? JSON.parse(tl.getVariable("agent.proxybypasslist")) : null;
-        const shouldProxyBypass = agentProxyBypassHosts?.includes(new URL(authorityURL).host);
+        let shouldProxyBypass = false;
+
+        const authorityHost = new URL(authorityURL).host;
+        for(var i=0; agentProxyBypassHosts && i < agentProxyBypassHosts.length; i++) {
```

Comment:
nit: You can use `Array.some` to replace `for` cycle. See details [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some).